salt/orchestrate/aws/security_groups.sls

```diff
@@ -163,6 +163,10 @@ create_rabbitmq_security_group:
           from_port: 25672
           to_port: 25672
           source_group_name: rabbitmq-{{ ENVIRONMENT }}
+        - ip_protocol: tcp
+          from_port: 35672
+          to_port: 35682
```

Comment:
Shouldn't this be `35672` since we only made the cli use a single ports as opposed to the range?

Reply:
No, because by default it uses a range and I removed the min/max from the env file. I had been experimenting with that before I discovered that it was a network issue.

Reply:
But in `pillar/rabbitmq/init.sls` used the port_min and port_max as 35672. And sounds like in their docs, u can restrict it to one port if u don’t have mutliple cli’s making queries, no?

Reply:
You can, but it's not required as long as the internode communication is working. Also, we often run the same command against all of the instances in a cluster so restricting the port range at the config and network layer may cause contention in those situations.